### ISSUE TITLE
landing page

### ISSUE THREAD

**Issue by cappamuzan123** [boney-10/FrankHub#7]
indecisive with the outlook of the landing page

**Comment by boney-10** [comment]
Yeah, I can understand that but we can concentrate more on functionality than design because investors always look for the best business ideas/ logic!
